internal/consts/consts.go

```diff
@@ -7,34 +7,42 @@ const (
 	/*
 		common field names
 	*/
-	FieldBindDN                     = "binddn"
-	FieldBindPass                   = "bindpass"
-	FieldCertificate                = "certificate"
-	FieldClientTLSCert              = "client_tls_cert"
-	FieldClientTLSKey               = "client_tls_key"
-	FieldDistinguishedNames         = "distinguished_names"
-	FieldUPNDomain                  = "upndomain"
-	FieldStartTLS                   = "starttls"
-	FieldConnectionTimeout          = "connection_timeout"
-	FieldRequestTimeout             = "request_timeout"
-	FieldSchema                     = "schema"
-	FieldPasswordPolicy             = "password_policy"
-	FieldLength                     = "length"
-	FieldInsecureTLS                = "insecure_tls"
-	FieldURL                        = "url"
-	FieldUserAttr                   = "userattr"
-	FieldUserDN                     = "userdn"
-	FieldRotationPeriod             = "rotation_period"
-	FieldPath                       = "path"
-	FieldPaths                      = "paths"
-	FieldParameters                 = "parameters"
-	FieldMethod                     = "method"
-	FieldNamespace                  = "namespace"
-	FieldNamespaceID                = "namespace_id"
-	FieldNamespacePath              = "namespace_path"
-	FieldBackend                    = "backend"
-	FieldPathFQ                     = "path_fq"
-	FieldData                       = "data"
+	FieldBindDN             = "binddn"
+	FieldBindPass           = "bindpass"
+	FieldCertificate        = "certificate"
+	FieldClientTLSCert      = "client_tls_cert"
+	FieldClientTLSKey       = "client_tls_key"
+	FieldDistinguishedNames = "distinguished_names"
+	FieldUPNDomain          = "upndomain"
+	FieldStartTLS           = "starttls"
+	FieldConnectionTimeout  = "connection_timeout"
+	FieldRequestTimeout     = "request_timeout"
+	FieldSchema             = "schema"
+	FieldPasswordPolicy     = "password_policy"
+	FieldLength             = "length"
+	FieldInsecureTLS        = "insecure_tls"
+	FieldURL                = "url"
+	FieldUserAttr           = "userattr"
+	FieldUserDN             = "userdn"
+	FieldRotationPeriod     = "rotation_period"
+
+	// FieldPath should be used for all new backend-mounting resources.
```

Comment:
nit: could we move these documented fields to the top and move everything else below them?